djautotask/sync_rest.py

```diff
@@ -129,6 +135,20 @@ def get_single(self, *args, **kwargs):
     def _assign_field_data(self, instance, api_instance):
         raise NotImplementedError
 
+    def _parse_date_time(self, instance, attribute_name):
```

Comment:
I think this method could be simplified and cleaned up a bit. For a start, if the instance that got passed is having an attribute set on it do we really need to return the instance? I would expect the updated attribute to already be set in `_assign_field_data` after setattr is run.

It also seems like this method does more than just parse a datetime - it also sets the attributes on the object.

Reply:
Actually do we need this method at all?

I might be missing something obvious but could we do something like:
```python
 first_response_date_time = json_data.get('firstResponseDateTime')
 if first_response_date_time:
     instance.first_response_date_time = parse(first_response_date_time)
```
in the `_assign_field_data` method?

Reply:
Good point as well! I was a little bit lazy to check if we actually need `return instance`. 

However, I think this method is useful for further usage. Comparing to SOAP API, which returns datetime object, REST_API returns string type, so in a future API method with a datetime info string we need to parse, which could our codes double or more. So I feel some function could be useful at this moment.

I also changed method name to `_set_datetime_attribute`.

Finally, I think we need to parse whether `if first_response_date_time:` is true of not. i.e. it can be set None or some value. Therefore, the feature of function is slightly different from your code snippets. :)

Reply:
Yeah fair enough. The name change is an improvement and it makes the code look a little cleaner.